components/CaseStatus/AddCaseStatusForm/AddCaseStatusForm.tsx

```diff
@@ -6,20 +6,52 @@ import FlexibleField from 'components/FlexibleForms/FlexibleFields';
 import Button from 'components/Button/Button';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
+import { useFormValues } from 'utils/api/caseStatus';
+import { Choice, Field } from 'data/flexibleForms/forms.types';
 
 const AddCaseStatusForm: React.FC<{
   personId: number;
   prefilledFields: any;
 }> = ({ personId, prefilledFields }) => {
   const router = useRouter();
-  const form_fields = CASE_STATUS.steps[0].fields;
+
+  const form_fields = [...CASE_STATUS.steps[0].fields];
```

Comment:
```react_devtools_backend.js:4049 Warning: Encountered two children with the same key, `startDate`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.```

there is a shared id meaning the key is being shared I think

Reply:
I think this was because we duplicated the start date form field so that it appear when you selected `CIN` or `CP`.

The start date field should appear if you selected either `CIN` and `CP`.

We duplicated the field as a work around because the start date form field had a condition set to appear only when you select a`CIN` and the conditions field on form fields doesn't handle OR operations. So essentially we override it depending on what was clicked. 

See [slack thread](https://hackit-lbh.slack.com/archives/C021Q88C953/p1632153326209500) for the conversation that inspired this but I'm starting to wonder if the start date needs to hide before a case status option is selected. 🤔 Maybe it can always been shown?